patroni/log.py

```diff
@@ -13,7 +13,14 @@
 from queue import Queue, Full
 from threading import Lock, Thread
 
-from typing import Any, Dict, List, Optional, Union, TYPE_CHECKING
+from typing import Any, Dict, List, Optional, Union, TYPE_CHECKING, TextIO
+
+if TYPE_CHECKING:
+    type_log_handler = Union[RotatingFileHandler, logging.StreamHandler[TextIO]]
+else:
+    type_log_handler = Union[RotatingFileHandler, logging.StreamHandler]
+
+type_logformat = Union[List[Union[str, Dict[str, Any], Any]], str, Any]
```

Comment:
Because pylance give me some errors. This is just one of them:

```Type of "field" is unknown Pylance(reportUnknownVariableType)```

```py
for field in logformat:
    if isinstance(field, str):
        log_fields.append(field)
    elif isinstance(field, dict):
        for original_field, renamed_field in field.items():
            if isinstance(renamed_field, str):
                log_fields.append(original_field)
                rename_fields[original_field] = renamed_field
            else:
                _LOGGER.warning('Expected renamed log field to be a string, but got "%s"', renamed_field)
```